apiary.apib

```diff
@@ -3250,6 +3250,357 @@ Update the current timer. Only possible if there is a timer running.
 
 + Response 204
 
+# Group Work Orders
+
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + performed_at: `2017-05-26T10:01:49+00:00` (string, required)
```

Comment:
Looking at some online work order templates I find things like "expected start date", "starting date", "scheduled start". Often in combination with the actual date on completion.

Reply:
This is renamed to `starts_at`.